### PR TITLE
Mention the 'release' vs postdeploy entry in heroku compatibility page

### LINKED ISSUE #380: In Heroku page, mention the postdeploy hook and its differences with the Heroku "release"

### PR DESCRIPTION
**Pull request by Soulou**
Fixes #380

### CODE REVIEW

File: _posts/platform/getting-started/2000-01-01-heroku-compatibility.md [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
-If a Procfile is available in the source of your app, it will be used. See the [dedicated Procfile page](/internals/procfile.html) for more informations.
+If a **Procfile** is available in the source of your app, it will be used. See the
+[dedicated Procfile page](/internals/procfile.html) for more informations.
+
+If a `release` entry is defined in your **Procfile**, you should use our
+[postdeploy hook]({% post_url platform/app/2000-01-01-postdeploy-hook %})
+instead, the *release* entry would not be executed. The difference is that the
+*postdeploy* hook is only getting executed at the end of a successful
+deployment, not at each change of variable/addon modification (creating a
+release)
```
Comment (EtienneM) [inline _posts/platform/getting-started/2000-01-01-heroku-compatibility.md]: Missing final dot

File: _posts/platform/getting-started/2000-01-01-heroku-compatibility.md [hunk 2]
```diff
@@ -1,23 +1,39 @@
...
 
 ## Buildpacks
 
-Most buildpacks and their functionalities work exactly the same as on Heroku. Some buildpacks are written and maintained by our own. It's open source. Check your [GitHub account](https://github.com/Scalingo/?query=buildpack) to see the whole list.
+Most buildpacks and their functionalities work exactly the same as on Heroku.
+Some buildpacks are written and maintained by our own. It's open source. Check
+your [GitHub account](https://github.com/Scalingo/?query=buildpack) to see the
```
Comment (EtienneM) [inline _posts/platform/getting-started/2000-01-01-heroku-compatibility.md]: ~your~ -> our

File: _posts/platform/getting-started/2000-01-01-heroku-compatibility.md [hunk 3]
```diff
@@ -1,23 +1,39 @@
 ---
 title: Heroku Compatibility
-modified_at: 2015-09-17 21:00:00
+modified_at: 2017-09-10 21:00:00
```
[the remark this answers is not in the document]
Comment (Soulou) [reply _posts/platform/getting-started/2000-01-01-heroku-compatibility.md]: Ahaha I don't know =D

File: _posts/platform/getting-started/2000-01-01-heroku-compatibility.md [hunk 4]
```diff
@@ -1,23 +1,40 @@
 ---
 title: Heroku Compatibility
-modified_at: 2015-09-17 21:00:00
+modified_at: 2019-09-10 21:00:00
```
Comment (yannski) [inline _posts/platform/getting-started/2000-01-01-heroku-compatibility.md]: 2018 would be better

**Pull request merged**